Language-Version-History.md

```diff
@@ -111,11 +112,23 @@ Features Added in C# Language Versions
 # [C# 1.0](https://en.wikipedia.org/wiki/Microsoft_Visual_Studio#.NET_.282002.29) - Visual Studio .NET 2002
 - Classes
 - Structs
+- Enums
 - Interfaces
 - Events
+- [Operator overloading](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/operator-overloading)
```

Comment:
In all cases, you should remove the `en-us`. (I'm only writing the suggestion on this instance). That way, the link will open in the reader's current culture:

```suggestion
- [Operator overloading](https://docs.microsoft.com/dotnet/csharp/language-reference/operators/operator-overloading)
```